package.json

```diff
@@ -151,11 +151,20 @@
   "release": {
     "branches": [
       "master",
-      "next",
+      {
+        "name": "next",
```

Comment:
this next change is still experimental as part of my ticket, can we omit it from this bramch?

Reply:
@Cyberxon Can we merge it now or is it still in progress?

Reply:
Thank you for the reminder, I just created a quick PR to fix that, feel free to approve and merge it so this will disappear